var/spack/repos/builtin/packages/iegenlib/package.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+# -----------------------------------------------------------------------------
+# Author: Derick Huth <derick.huth@utah.edu>
+# -----------------------------------------------------------------------------
+#   Note: This package was created by the ctop reaserch group at
+#         the University of Utah, Shool of Computing and is unifiliated
+#         with IEGenLib
+# -----------------------------------------------------------------------------
```

Comment:
Is this message needed?  It's not typical for Spack.  The Git Repo and Spack history already shows who created it; but over time, it's likely that others might add to / change the package significantly.

Reply:
Will remove it then.